src/solver/page/request-drawer/SolverLimitationsDrawer.tsx

```diff
@@ -1,84 +1,167 @@
                 label={
                   <Group gap="xs">
-                    <Image
-                      src={building.imagePath.replace('_256', '_64')}
-                      width={24}
-                      height={24}
-                    />
-                    {building.name}
+                    <FactoryItemImage size={24} id={resource} />
+                    {item.name}
                   </Group>
                 }
-                checked={!request?.blockedBuildings?.includes(building.id)}
+                checked={!request?.blockedResources?.includes(resource)}
                 onChange={e =>
                   useStore
                     .getState()
-                    .toggleBlockedBuilding(
+                    .toggleBlockedResource(
                       id!,
-                      building.id,
+                      resource,
                       !e.currentTarget.checked,
                     )
                 }
               />
-            ))}
-          </Stack>
-        </SimpleGrid>
-      </Stack>
-    </>
+            );
+          })}
+        </Stack>
+        <Stack gap="xs">
+          <Text size="lg">Buildings</Text>
+          {FactoryBuildingsForRecipes.map(building => (
+            <Checkbox
+              key={building.id}
+              label={
+                <Group gap="xs">
+                  <Image
+                    src={building.imagePath.replace('_256', '_64')}
+                    width={24}
+                    height={24}
+                  />
+                  {building.name}
+                </Group>
+              }
+              checked={!request?.blockedBuildings?.includes(building.id)}
+              onChange={e =>
+                useStore
+                  .getState()
+                  .toggleBlockedBuilding(
+                    id!,
+                    building.id,
+                    !e.currentTarget.checked,
+                  )
+              }
+            />
+          ))}
+        </Stack>
+        <Stack gap="sm" style={{ gridColumn: 'span 2' }}>
+          <Text size="lg">Logistics</Text>
+          <Text size="sm">
+            Selecting a belt or a pipeline will highlight in the calculator all
+            the ingredients which are exceeding the belt / pipeline amount.
+          </Text>
+        </Stack>
+        <Stack gap="xs">
+          <Text size="lg">Belt Logistic</Text>
+          {FactoryConveyorBelts.map((building, index) => (
+            <Radio
+              key={building.id}
+              label={
+                <Group gap="xs">
+                  <Image
+                    src={building.imagePath.replace('_256', '_64')}
+                    width={24}
+                    height={24}
+                  />
+                  {building.name}
+                </Group>
+              }
+              checked={
+                maxBelt === building.id ||
+                (!maxBelt && index === FactoryConveyorBelts.length - 1)
```

Comment:
It's still possible to have _no_ belts selected (it's useful if the user doesn't want to highlight any edge in the graph), so we need to handle that. With the select it would be free, with the radio we should have a way to unselect it (like another radio option?)

Reply:
Yeah, if you have nothing set the last option is displayed, but not persisted. The calculation is not affected per se, if no option is set. We can change it, so that no selected state is displayed if no option is set.

Reply:
Yup calculation is the same for the belts amount display, but it doesn't get _highlighted_ (the brownish color the edge label gets when it's over the tooltip)

We could add an option "Do not highlight belt" but I don't like it, just showing the idea. Maybe just "No belt"?

Reply:
My personal opinion, it's fine as is. I would not add the "no belt" function, because imo it would just make the whole thing more complicated, usability wise.
The other direction would be, to add a gaming setting to turn the highlight on and off, hence making the whole process more consistent. I guess mixing up the display of the belts / pipeline, and setting max limits as a highlight constraints aim to achieve two different things. That being said, it's your decision how we tackle that.

Reply:
I was thinking about the second option too after my comment, I think you're right here. Having the two options mixed is just strange. Let's merge this in the meantime